spec/better_errors/error_page_spec.rb

```diff
@@ -88,5 +88,35 @@ def initialize(message = nil)
         expect(error_page.exception_message).not_to match(/\A\n\n/)
       end
     end
+
+    it "shows variables with inspects that are below the inspect size threshold" do
+      BetterErrors.maximum_variable_inspect_size = 50_000
+
+      content = 'AAAAA'
+      empty_binding.instance_variable_set('@small', content)
+
+      html = error_page.do_variables("index" => 0)[:html]
+      html.should_not include "object too large"
+    end
+
+    it "hides variables with inspects that are above the inspect size threshold" do
+      BetterErrors.maximum_variable_inspect_size = 50_000
+
+      content = 'A' * (BetterErrors.maximum_variable_inspect_size)
+      empty_binding.instance_variable_set('@big', content)
+
+      html = error_page.do_variables("index" => 0)[:html]
+      html.should include "object too large"
+    end
+
+    it "shows variables with large inspects if max inspect size is disabled" do
+      BetterErrors.maximum_variable_inspect_size = nil
+
+      content = 'A' * (50_000)
```

Comment:
Will be nice to test if this is over default (so default is overridden by `nil`).